LibreNMS/Modules/OS.php

```diff
@@ -102,7 +103,18 @@ private function handleChanges(\LibreNMS\OS $os)
     private function updateLocation(\LibreNMS\OS $os)
     {
         $device = $os->getDevice();
-        if ($device->override_sysLocation) {
+
+        if (\LibreNMS\Config::get('parse_dns_location_record')) {
```

Comment:
This code should go inside the lookupCoordinates() method

Reply:
ok, so it goes into Model ... are you sure?
i had it in this method first and then i had to move it to OS ... now back to Model?

Reply:
@SourceDoctor probably model not the best place for the method, but it belongs inside that method.

Reply:
@murrant moved :-)